fido/fido.py

```diff
@@ -34,8 +34,8 @@ def _import_reactor():
 
 
 def _twisted_web_client():
```

Comment:
Yes, avoiding to import fido.client at the module level. There are quite a few places where the function is used and we probably wouldn't want to litter the file with `from fido.client import _twisted_web_client`